### PR TITLE
Add master field to account context

### LINKED ISSUE #71: When creating new account, derive from master by default.
For now, let's add 3rd button. 
Assume Master account is simply the first account.

Generate new derivation paths. //0 //1 //2 ...

### PR DESCRIPTION
**Pull request by VladStarostenko**
closes #71

### CODE REVIEW

File: packages/extension-ui/src/Popup/Accounts/index.tsx [hunk 1]
```diff
@@ -4,27 +4,17 @@
...
 import AccountsTree from './AccountsTree';
 import { AccountContext, Button, Header, MediaContext, AddAccount, ButtonArea, Svg, ButtonWithSubtitle } from '../../components';
 
 type Props = {};
 
-function findMasterAccount (accounts: AccountJson[]): AccountJson | undefined {
-  return accounts
-    .map((account) => ({ ...account, whenCreated: account.whenCreated || Infinity }))
-    .sort((a, b) => a.whenCreated - b.whenCreated)
-    .find((account) => !account.isExternal);
```
Comment (krzysztof-jelski) [inline packages/extension-ui/src/Popup/Accounts/index.tsx:19]: Shouldn't we move filtering out external acocunts to `buildHierarchy` now?

Comment (vanruch) [reply packages/extension-ui/src/Popup/Accounts/index.tsx:19]: That's why those lines are removed :)

Comment (krzysztof-jelski) [reply packages/extension-ui/src/Popup/Accounts/index.tsx:19]: But in `buildHierarchy` there's no filter for external.

Comment (vanruch) [reply packages/extension-ui/src/Popup/Accounts/index.tsx:19]: Yep, you are right, missed that

**Review by krzysztof-jelski — APPROVED** [review]

**Pull request merged**